adapter/src/hermes/adapter/constants.h

```diff
@@ -0,0 +1,20 @@
+#ifndef HERMES_ADAPTER_CONSTANTS_H
+#define HERMES_ADAPTER_CONSTANTS_H
+
+/**
+ * Constants file for all adapter.
+ * This file contains common constants across different adapters.
+ */
+
+/**
+ * kHermesConf env variable is used to define path to kHermesConf in adapters.
+ * This is used for initialization of Hermes.
+ */
+const char* kHermesConf = "kHermesConf";
```

Comment:
Sorry, it makes sense for the environment variable to be named `HERMES_CONF` since that's standard for environment variables. I just meant the identifier should follow Google's conventions.

Reply:
I see fixed it as well.